src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -7,7 +7,7 @@ type WalletInvoice = {
 }
 
 type WalletInvoiceValidator = {
-  validateToSend({ fromWalletId: WalletId }): true | ApplicationError
+  validateToSend({ fromWalletId }: { fromWalletId: WalletId }): true | ApplicationError
```

Comment:
this should not be an object. (one param rule)

```suggestion
  validateToSend(fromWalletId: WalletId): true | ApplicationError
```